README.md

```diff
@@ -18,3 +18,20 @@ The project is open source under the GNU Affero General Public License, with a p
 - Go to `https://tools.wmflabs.org/wikidata-todo/quick_statements.php`.
 - Authenticate yourself with your Wikidata account.
 - Copy and paste the contents of `quickstatement_commands.txt` into the blank field, and run the commands
+
+## Field mappings
+- bezeichnung -> P...
+- brunnenart_txt -> P...
+- wasserart_txt -> P...
+- nummer -> P...
+- historisches_baujahr -> P...
+
+## Contributions welcome
+The WVZ (https://www.wikidata.org/wiki/Q53629101) does not have all that often a change in its base data. Nonetheless does it make sense to cross-check their master-data-set with what is in WikiData on a regular basis.
+
+Therefore: 
+1) convert this into a Bot https://www.wikidata.org/wiki/Wikidata:Creating_a_bot , https://www.wikidata.org/wiki/Wikidata:Bots 
+2) create a list of differences where the value in wikidata and the value in the WVZ master are different
+3) in the case that community edits might be of higher quality than the WVZ master, prepare a list of suggested changes to the owner of the WVZ dataset (probably needs some human judgement/manual work)
```

Comment:
This is a very labour intensive approach, we should discuss alternatives. This discussion about the roadmap should be removed from the readme and take place in the form of issues. --> remove these points and create an issue for each, and just refer to open issues in this section